style_rwd.css

```diff
@@ -2,21 +2,233 @@
     img.main_photo {
         display: none;
     }
-    .catch_phrase{
-        max-width: 767px;
+    h2 {
+        margin-block-start: 0;
+        margin-block-end: 0;
```

Comment:
don't use `margin-block-end: 0;` you can achieve same result setting `margin: 0`

Reply:
fixed